subscription_oca/models/sale_subscription.py

```diff
@@ -249,6 +249,16 @@ def action_close_subscription(self):
             "res_id": False,
         }
 
+    def close_subscription(self, close_reason_id=False):
+        self.ensure_one()
+        self.recurring_next_date = False
+        closed_stage = self.env["sale.subscription.stage"].search(
```

Comment:
I don't really like selection with a limit=1. Is there a mean to restrict stage selection in a better way ?

Reply:
Propose a better solution

Reply:
Nevermind, don't bother, this will never be merged anyways...